src/main/java/edu/ie3/datamodel/models/input/container/ThermalGrid.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.models.input.container;
+
+import edu.ie3.datamodel.models.input.thermal.ThermalBusInput;
+import edu.ie3.datamodel.models.input.thermal.ThermalHouseInput;
+import edu.ie3.datamodel.models.input.thermal.ThermalInput;
+import edu.ie3.datamodel.models.input.thermal.ThermalStorageInput;
+import java.util.*;
+
+/**
+ * Container object to denote a fully connected thermal "grid". As there are currently no branch
+ * elements, a grid always only consists of one {@link ThermalBusInput} and all its connected {@link
+ * edu.ie3.datamodel.models.input.thermal.ThermalUnitInput}s
+ */
+public record ThermalGrid(
+    ThermalBusInput bus, Set<ThermalHouseInput> houses, Set<ThermalStorageInput> storages)
```

Comment:
Do you think it'd pay off to split thermal storages into their subtypes CylindricalStorage, ...?